composer.json

```diff
@@ -21,14 +21,17 @@
     ],
     "require": {
         "php": "^8.1",
-        "silverstripe/framework": "^5",
+        "silverstripe/framework": "^5.3",
         "silverstripe/versioned": "^2",
         "guzzlehttp/guzzle": "^7"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.6",
         "squizlabs/php_codesniffer": "^3"
     },
+    "conflict": {
+        "cwp/cwp-core": "<3.1"
```

Comment:
Added so that https://github.com/silverstripe/cwp-core/pull/134 is used.

Possibly is already broken in test mode without that PR, though I think it's best to ensure keep the conflict